### DISCUSSION TITLE
Duda de como hacer Planeación de la campaña publicitaria

### DISCUSSION THREAD

**Discussion by J0SE-ESC0BAR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#350] [Q&A] [answered]
Hola, no se como es que se requiere la Planeación de la campaña publicitaria, si es de hacer una planeación  con miro o de hacer en si la campaña publicitaria, como con imágenes de vacunación con canva
![Publicidad](https://user-images.githubusercontent.com/74581066/157868299-d700308a-ea9a-492d-8cfe-e1217ce57635.png)
.
![Planeación](https://user-images.githubusercontent.com/74581066/157867473-69bdf232-11da-426a-a69e-a2a901e6868b.png)

**Comment by doguedogue** [comment]
Creo que alguien de marketing nos lo respondería mejor.

Pero yo que solo soy un ser humano, lo que haría sería radio, tv, YouTube, tik tok, facebook, etc.

Diseñar  la campaña alusiva a mi producto o servicio.

**Comment by AngelCruzO** [comment, marked as answer]
Con lo poco que se de campañas publicitarias, se supone que hay que diseñar como bien dice @doguedogue, como de canales de comunicación pasan al sitio web.

Ejemplo: se publica un video al dia en facebook con el contenido de vacunación para jovenes de 18 a 25 años, que sea llamativo y con el enlace al sitio web. Ya dentro del sitio, tendran que realizar las acciones importantes, en este caso agendar una cita para vacunarse, posteriormente se finaliza la captación del usuario o se realiza un flujo mas para las siguientes dosis.

En otras palabras, es realizar un flujo que se debe realizar para captar usuarios y que acciones se deben realizar para que haga las acciones que se deseen o no. 

[Ejercio de plan publicitario](https://miro.com/app/board/uXjVOGmpBL0=/?invite_link_id=651295536913)

**Reply by J0SE-ESC0BAR** [reply]
Lo que e hice es esto pero no se si era esto o era hacer en si hacer la publicidad con imágenes 
![image](https://user-images.githubusercontent.com/74581066/158000835-292abd59-e5e6-42e7-b678-823d7e4c70fc.png)

**Reply by AngelCruzO** [reply]
Si esta bien, cada quien lo hará de manera diferente, Pero el chiste es que sepas como vas a distribuir la información

**Reply by J0SE-ESC0BAR** [reply]
Gracias, tenia esa duda

**Reply by GloriaCoralCerecedo** [reply]
Me gusto tu esquema, creo que usare los mismos subgrupos
